src/components/inbox/media-gallery/MediaPreviewDialog.tsx

```
@@ -11,36 +12,42 @@ interface MediaPreviewDialogProps {
 }
 
 export function MediaPreviewDialog({ item, open, onOpenChange }: MediaPreviewDialogProps) {
-  const { url: resolvedUrl, isLoading, error, refresh } = useResolvedStorageUrl(item?.url || '');
-  if (!item) return null;
+  // Keep last non-null item so Dialog exit animation has content to display
+  const lastItemRef = useRef<MediaItem | null>(null);
+  if (item) lastItemRef.current = item;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/components/inbox/media-gallery/MediaPreviewDialog.tsx:17:13
  15 |   // Keep last non-null item so Dialog exit animation has content to display
  16 |   const lastItemRef = useRef<MediaItem | null>(null);
> 17 |   if (item) lastItemRef.current = item;
     |             ^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  18 |   const displayItem = item ?? lastItemRef.current;
  19 |
  20 |   const { url: resolvedUrl, isLoading, error, refresh } = useResolvedStorageUrl(displayItem?.url || '');

(react-hooks/refs)